src/java_binding/src/lib.rs

```diff
@@ -217,7 +261,7 @@ pub extern "system" fn Java_com_risingwave_java_binding_Binding_vnodeCount(
 }
 
 #[no_mangle]
-pub extern "system" fn Java_com_risingwave_java_binding_Binding_iteratorNew<'a>(
+pub extern "system" fn Java_com_risingwave_java_binding_Binding_hummockIteratorNew<'a>(
```

Comment:
Since the java side is renamed to `HummockIterator`, should the rust side get renamed also?

Reply:
renamed to `HummockJavaBindingIterator`